app/models/cloud_network.rb

```diff
@@ -40,7 +38,7 @@ class CloudNetwork < ApplicationRecord
     end
   end
 
-  virtual_total :total_vms, :vms
+  virtual_total :total_vms, :vms, :uses => :vms
```

Comment:
Pretty sure this defaults `:uses` to the relation. So it should pick up `:vms` automatically.